cmd/dep/ensure.go

```diff
@@ -127,7 +128,18 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 	if err != nil {
 		return errors.Wrap(err, "ensure ListPackage for project")
 	}
-
+	if len(params.RootPackageTree.Packages) == 0 {
+		return errors.New("Current directory doesn't contain any go code")
+	}
+	for impPth, pckOrMap := range params.RootPackageTree.Packages {
```

Comment:
nit: let's use `poe` instead of `pckOrMap` - a simple acronym for `PackageOrErr`